InnoClinic.OfficesAPI/BLL/Helpers/ParseIdHelper.cs

```diff
@@ -0,0 +1,14 @@
+﻿using BLL.Exceptions;
+using MongoDB.Bson;
+
+namespace BLL.Helpers;
+
+public static class ParseIdHelper
```

Comment:
It will be better to name it `ParseHelper` instead of `ParseIdHelper` to make it more scalable